x-pack/plugins/session_view/public/components/process_tree_node/styles.ts

```diff
@@ -100,6 +100,18 @@ export const useStyles = ({ depth, hasAlerts, hasInvestigatedAlert }: StylesDeps
       color: colors.successText,
     };
 
+    const timeStamp: CSSObject = {
+      float: 'right',
+      fontFamily: font.familyCode,
+      fontSize: size.m,
+      fontWeight: font.weight.regular,
+      lineHeight: '18px',
```

Comment:
Is the lineHeight and letterSpacing necessary? Is it a matter of vertical alignment?

Reply:
I put it there as per the specs Max gave to me

Reply:
if we can avoid hardcoding values lets do that, i would only set line-height if it's needed.